### DISCUSSION TITLE
flutter Listview.seperated, fetching data

### DISCUSSION THREAD

**Discussion by hhamja** [supabase/supabase#7674] [Questions]
I'm using Supabase + Flutter and I'm trying to return a Listview.builder using a photo-like datatype and Future.builder Widget.
<img width="400" alt="123" src="https://user-images.githubusercontent.com/100702371/178779314-d4d5417d-70bf-4fff-89c6-f9621d1b9432.png">

How to fetching title data and username data in Listview.builder in order? It's a very difficult task for me...

**Comment by dshukertjr** [comment]
Hi @CodeKinggggggg,

Could you provide answers to these questions as a reply to this post to further assist you?
- Do you have any code that you currently have?
- What is "photo-like data type"?
- If you are using Listview.builder, what is it a list of?
